assets/js/theme/common/aria/radioOptions.js

```diff
@@ -40,12 +41,14 @@ const handleItemKeyDown = itemCollection => e => {
     case ariaKeyCodes.UP: {
         const prevItemIdx = calculateTargetItemPosition(lastCollectionItemIdx, itemIdx - 1);
         itemCollection.get(prevItemIdx).focus();
+        setCheckedRadioItem(itemCollection, itemIdx - 1);
```

Comment:
If we are moving to instant option selection, is there is an ability to remove SPACE and Enter switch case

Reply:
yes, I believe it's safe to remove them